test/index.js

```diff
@@ -190,3 +192,27 @@ test("works with no styles at all", function(t) {
 
   t.end()
 })
+
+test("works with caching", function(t) {
+  var opts = {path: importsDir, cacheDir: cacheDir}
+  var cssFileName = path.resolve("test/fixtures/relative-to-source.css")
+  var postcssOpts = {from: cssFileName}
+  var css = fs.readFileSync(cssFileName)
+  var output = postcss()
+    .use(atImport(opts))
+    .process(css, postcssOpts)
+    .css.trim()
+
+  var imports = JSON.parse(fs.readFileSync(cacheDir + "/imports.json"))
+  var cacheFileName = imports[cssFileName].cache;
+  var cache = fs.readFileSync(cacheFileName, "utf8").trim()
+
+  t.equal(output, cache, "should put output in cache")
```

Comment:
I'm sorry @MoOx, there's no way of knowing. It's a transparent cache. Unless you're suggesting that we should add a bunch of debug code to poll the internal state of the engine, which I would strongly advice against, we might as well close the PR because there's no way for me to satisfy this need. I know that the cache is hit by using breakpoints, I know that it returns a correct and consistent result, and I know that I get a 100ms performance increase when using it with a local project of mine, but there's no way for me to put that into a unit test.